roottest/root/tree/cache/Event.h

```diff
@@ -1,204 +0,0 @@
-#ifndef ROOT_Event
-#define ROOT_Event
-
-//////////////////////////////////////////////////////////////////////////
-//                                                                      //
-// Event                                                                //
-//                                                                      //
-// Description of the event and track parameters                        //
-//                                                                      //
-//////////////////////////////////////////////////////////////////////////
-
-#include "TObject.h"
-#include "TClonesArray.h"
-#include "TRefArray.h"
-#include "TRef.h"
-#include "TH1.h"
-#include "TBits.h"
-#include "TMath.h"
-
-
-class TDirectory;
-
-class Track : public TObject {
-
-private:
-   Float_t      fPx;           //X component of the momentum
-   Float_t      fPy;           //Y component of the momentum
-   Float_t      fPz;           //Z component of the momentum
-   Float_t      fRandom;       //A random track quantity
-   Float16_t    fMass2;        //[0,0,8] The mass square of this particle
```

Comment:
All 4 removed Event.h have significant differences (for example here `fMass2` has a different type).  How were those differences handled/preserved?

Reply:
All 4 seemed to have each their own quirk/changes.  Did you catalog and verify that each of those changes were 'innocuous'?

Reply:
After checking, the difference indeed result in basket size differences that lead to a different layout on the file and thus difference fetching by the cache.  I agree that the difference is spurious in regard to this test.